src/approaches/grammar_search_invention_approach.py

```diff
@@ -360,6 +363,87 @@ def enumerate(self) -> Iterator[Tuple[Predicate, float]]:
                     yield (uff_predicate, cost)
 
 
+### Useful for debugging ###
+
+# # Replace these strings with anything you want to exclusively enumerate.
```

Comment:
`# #` -> `#` in these 3 lines

Reply:
But then if you block uncomment, you have to re-comment these lines